### ISSUE TITLE
Request for fix: false positive AvoidConcatInAppend

### ISSUE THREAD

**Issue by stokpop** [jborgers/PMD-jPinpoint-rules#79]
This code has a false positive in AvoidConcatInAppend:

```java
    static String foo(String text) {
        StringBuilder builder = new StringBuilder(); 
        for (int i = 0; i < text.length(); i++) { // rule violation points to this line (wrong line)
            char ch = text.charAt(i);
            String bar = String.valueOf(ch - 65 + 10);  // false positive
            builder.append(bar);
        }
        return builder.toString();
    }
```

**Comment by jborgers** [comment]
Fixed
